ApplicationServices/LibraryCommands.cs

```diff
@@ -161,37 +154,22 @@ private static async Task<int> importIntoDbAsync(List<ImportItem> importItems)
 		}
 		#endregion
 
-		#region Update book details
-		public static int UpdateUserDefinedItem(Book book, string newTags, LiberatedStatus bookStatus, LiberatedStatus? pdfStatus)
+		#region Update book details		
+
+		public static int UpdateTags(Book libraryBook, string newTags)
 		{
 			try
 			{
 				using var context = DbContexts.GetContext();
 
-				var udi = book.UserDefinedItem;
-
-				var tagsChanged = udi.Tags != newTags;
```

Comment:
The purity of this change is attractive. However we now have up to 3 discrete db calls -- if the user changes tags, book liberation, and pdf liberation. If we later add further editing on that form, each of those will also use a new db call as well as their own transaction.

Reply:
Since grid entry is now the only piece interacting with the database, I can expose Begin and End transaction methods to cut down on overhead.

Reply:
That would help. Or a Transaction Scope. But it's still 3 separate db calls rather than 1. Obviously 1 vs 3 is trivial, but I'm concerned that it establishes a pattern that will only grow with future expansion, rather than being 1 update call.

Reply:
OK, I added BeginEdit and EndEdit to GridEntry, and LibraryCommands only has a single method for updating the database for UserDefinedItem: UpdateUserDefinedItem.